packages/dds/tree/src/shared-tree-core/sharedTreeCore.ts

```diff
@@ -64,7 +64,7 @@ export interface ClonableSchemaAndPolicy extends SchemaAndPolicy {
  * TODO: is history policy a detail of what indexes are used, or is there something else to it?
  */
 export class SharedTreeCore<TEditor extends ChangeFamilyEditor, TChange> extends SharedObject {
-	private readonly editManager: EditManager<TEditor, TChange, ChangeFamily<TEditor, TChange>>;
+	protected readonly editManager: EditManager<TEditor, TChange, ChangeFamily<TEditor, TChange>>;
```

Comment:
Exposing the `editManager` to child classes seems unnecessarily heavy handed: there's a LOT of things that a child class could do with this `editManager` that would cause all manner of problems for this class. It looks like you only need to know the revision of the commit at the tip of the trunk, so I recommend just making a getter for that.